unit-10/main.js

```diff
@@ -0,0 +1,132 @@
+// bài 1: In một bảng nhân của một số bất kỳ với các số từ 1 tới 10 và hiển thị kết quả.
+const tableOfMultipliers = num => {
+  for (let i = 1; i <= 10; i++) {
+    console.log(num * i)
+  }
+}
+tableOfMultipliers(10)
+
+/* bài 2: Hàm nhận tham số là n (số nguyên dương),
+  1 <= n <= 30 In các số chẵn từ 1 đến n Vd n = 6: in ra: 2 4 6 */
+const getEven = num => {
```

Comment:
`getEvenNumber` thì sẽ rõ nghĩa hơn

```suggestion
const getEvenNumber = num => {
```